routes/cvApiRoutes.ts

```diff
@@ -17,12 +18,34 @@ module.exports = function(app: any) {
       res.send(JSON.parse(body))
     })
   })
+
+  // to create a cv
   app.post('/add', (req: any, res: any) =>{
-    request.post({url:'http://localhost:3000/cv',form:req.body}, function (error:any, response:any, body:any) {
+    request.post('http://localhost:3000/cv',{form:req.body}, function (error:any, response:any, body:any) {
+      console.log('error:', error);
+      console.log('statusCode:', response && response.statusCode);
+      console.log('body:', body);
+      res.send(JSON.parse(body))
+    })
+  })
+
+  // to see a specific cv
+  app.get('/list/:cvId', (req: any, res: any) =>{
+    request.get('http://localhost:3000/cv/' + req.params.cvId, function (error:any, response:any, body:any) {
       console.log('error:', error);
       console.log('statusCode:', response && response.statusCode);
       console.log('body:', body);
       res.send(JSON.parse(body))
     })
   })
-};
+
+    // to edit a specific cv (not being able to update, needs to be fixed)
+    app.put('/edit/:cvId', (req: any, res: any) =>{
+      request.put('http://localhost:3000/cv/' + req.params.cvId, function (error:any, response:any, body:any) {
```

Comment:
Why do you have 2 request initialisation here?

Reply:
one is for this `proxy-api` and one is for requesting from the `cv-api`